src/periphery/contracts/misc/UiPoolDataProviderV3.sol

```diff
@@ -72,10 +72,10 @@ contract UiPoolDataProviderV3 is IUiPoolDataProviderV3 {
       //the current variable borrow rate. Expressed in ray
       reserveData.variableBorrowRate = baseData.currentVariableBorrowRate;
       //the current stable borrow rate. Expressed in ray
-      reserveData.stableBorrowRate = baseData.currentStableBorrowRate;
```

Comment:
is `ReserveDataLegacy` struct updated in 3.2? wondering if this will be compatible

Reply:
No it's the same in 3.2 for backwards compatibility